front/app/containers/Admin/projects/project/general/index.tsx

```diff
@@ -565,12 +569,22 @@ const AdminProjectsProjectGeneral = () => {
           imageUrl={project?.attributes.header_bg.large}
           onImageChange={handleHeaderBgChange}
         />
-
-        <ProjectImageDropzone
-          projectImages={projectImages}
-          handleProjectImagesOnAdd={handleProjectImagesOnAdd}
-          handleProjectImageOnRemove={handleProjectImageOnRemove}
-        />
+        {projectCardShouldBeSaved ? (
+          <Box display="flex" flexDirection="column" gap="8px">
+            <ImageCropperContainer
+              image={projectCard}
+              onComplete={handleProjectCardOnCompleteCropping}
+              aspect={CARD_IMAGE_ASPECT_RATIO / 1}
+              onRemove={handleCroppedProjectCardOnRemove}
+            />
+          </Box>
+        ) : (
+          <ProjectImageDropzone
+            projectImages={projectCard && [projectCard]}
```

Comment:
`projectCard ? [projectCard] : null` states the intention better imo. Wdyt?

Reply:
It's the same for me.

Reply:
With my version you see what the value will be set to without having to hover projectCard to see potential values. So not the same imo.